lib/zendesk_api/middleware/response/sanitize_response.rb

```diff
@@ -0,0 +1,21 @@
+module ZendeskAPI
+  module Middleware
+    module Response
+      class SanitizeResponse < Faraday::Response::Middleware
+        def on_complete(env)
+          to_utf8!(env[:body]) unless env[:body].strip.empty?
```

Comment:
force_encoding etc should also work fine on empty strings + (I assume) we rarely expect empty strings ... so no need to optimize for them :)